README.md

```diff
@@ -219,6 +219,30 @@ container.register(Animal.self) { _ in Cat(name: "Mimi") }
 
 If you implicitly instantiate `UIWindow` and its root view controller, the registrations setup for "Main" storyboard can be shared with the referenced storyboard since `defaultContainer` is configured in `setup` method.
 
+### Arguments
+if you want to pass some arguments, you can do it in such way:
+
+1) Register VC
+```swift
+let container = SwinjectStoryboard.defaultContainer
+container.storyboardInitCompleted(AnimalViewController.self) { r, c, arg1: Int, arg2: SomeValue in
+    c.animal = r.resolve(Animal.self)
+    c.countAnimals = arg1
+    c.someValue = arg2
+}
+container.register(Animal.self) { _ in Cat(name: "Mimi") }
+```
+
+2) Resolve VC
+```swift
+let sb = SwinjectStoryboard.create(
+    name: "Animals", bundle: nil, container: container)    
+let firstArg: Int = 5
+let secondArg: SomeValue = SomeValue()
+let catController = sb.instantiateViewController(withIdentifier: "SomeIdentifier", arg1: firstArg, arg2: secondArg) as! AnimalViewController
```

Comment:
It would be more consistent to have API similar to Swinject, i.e.
`instantiateViewController(withIdentifier: "SomeIdentifier", arguments: firstArg, secondArg)`